### DISCUSSION TITLE
2 or more pictures with block function in one row

### DISCUSSION THREAD

**Discussion by cracksilver** [automadcms/discussions#30] [General]
Hi there 
First I would like to say Automad is a phantastic flatcms. Thank you for all this work.
I have our company page wich I want to move to automad. So to do that I need 2 pics with text in a row next to each other.

After hours last night finding out to solve my problem I found the solution In this video on the top of the page: https://automad.org/user-guide/using-blocks 

Since I use standard v1 light theme I think this should works for me aswell. Here a screenshot what I'm missing in my backadmin/editor:
![screenshot_block_](https://github.com/user-attachments/assets/d77737cb-aae8-45af-b4fc-cca8341ede7f)

First thing I do:
![grafik](https://github.com/user-attachments/assets/eab21744-88ab-4848-9e42-735384f9220b)

next:
![grafik](https://github.com/user-attachments/assets/839015ba-3b85-4275-87e4-2aec0ceca2d4)

so I have one pic:
![grafik](https://github.com/user-attachments/assets/40b893e2-6c4c-43ed-ad1e-53781a1b7990)

And what to do now?
![grafik](https://github.com/user-attachments/assets/431d4175-2b5d-42e0-9b71-1f28f6b20123)

If I add a new picture the editor inserts it below the other picture.

What do I wrong? Is it not possible with this standard v1 light theme or am I just not smart enough?

thank you in advance.
Greg

by the way: I'm not a developer and I can't program. So the question is, are there developers who could help me with a few things? I still have more ideas ;-) of course I would also like to pay for it.

**Comment by cracksilver** [comment]
Hi again

Could it be that it has something to do with the version? I'm using version 2.0.0-alpha.8. The demo page is also 2.0.0-alpha.9. On the demo page, multiple boxes in sections don't work either.

Does the version that is on the demo video in the link above shows the old version 1.x? Were there other functions there? Or did I choose the wrong template? I have already tried dahmen and revitron. This function is missing there too.

It would be great if someone could show me where the problem is or show me what I'm doing wrong.

thank you
greg

**Comment by cracksilver** [comment]
I installed automad 1.10.9 now and it is working. 
Actually I wanted to start the new project with V2. Does anyone know when this function is in V2 available?

**Comment by marcantondahmen** [comment]
Hi there, below the filter in your last screenshot is a dropdown with "Automatische Breite". There you can change the width of any block inside a sction.

**Comment by cracksilver** [comment]
ahaaa....!
Thanks for that. I've never tried that out. Tried just the first two entries.... omg

For somebody looking for the same function:
![grafik](https://github.com/user-attachments/assets/2099ed32-25e0-4a07-a646-bd7f1dd2803b)

Danke und viele Grüsse nach Basel ;-)
Greg

**Reply by marcantondahmen** [reply]
Hey great you found it! Enjoy! 

Und Grüsse zurück!
